handbook/ce/customer-support-onboarding.md

```diff
@@ -1,5 +1,204 @@
+* Join the following developer communities to help make sure folks talking about Sourcegraph are getting their questions answered:
+	* Gopher Slack community. Post in #customer-support-chat so a member of the team can add you. You will need to join with your personal email address so our presence feels helpful and not sales-y. 
+* Read [Our ABC’s Always Be Coding](https://about.sourcegraph.com/blog/our-abcs-always-be-coding-childrens-book/). It’s just cute and so why not!? It’s also a great source of inspiration when trying to simplify complex ideas.
+
+
+## By the end of your second week, the following will be true...
+* You have helped at least 1 customer
+* You are participating in our team rituals
+* You can wrap your head around who are customers are
+* You feel confident that you know what Sourcegraph can do functionality wise, at a high level and in detail
+* You feel confident you know the ways that Sourcegraph can be configured and why
+* You feel confident you know the ways that Sourcegraph can be deployed and why
+
+### In order to ensure the above declarations are true, you will complete the following tasks:
+
+* Write your first weekly planning post (one of our [rituals](https://about.sourcegraph.com/handbook/ce/support-team-rituals)). You can see examples in our #customer-support-chat Slack channel.
+* Start contributing to the daily captain’s log thread (another of our [rituals](https://about.sourcegraph.com/handbook/ce/support-team-rituals)). You can see examples in our #customer-support-chat Slack channel.
+* Learn about who our customers are by checking out the [personas](https://about.sourcegraph.com/handbook/marketing/personas) overview and how Sourcegraph is useful for them by reading a few [case studies](https://about.sourcegraph.com/case-studies).
+* Explore the [product overview](https://about.sourcegraph.com/) and for each feature, explore the related [documentation](https://docs.sourcegraph.com/getting-started).
+* Spend some time getting familiar with [the Sourcegraph glossary](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/enterprise/docs/glossary.md)
+* Read the last few release blog posts on [our blog](https://about.sourcegraph.com/blog) and decide how you want to remember where to find this to look at later and often (when a customer has an issue, it’s possible it could be caused by something in a recent release)
+* Get a Sourcegraph instance [deployed](https://docs.sourcegraph.com/admin/install) using the `docker-compose` flow. This will be your own test environment. Watch [Dave’s Kubernetes Walkthrough video](https://drive.google.com/drive/folders/1JqQlrmGX8u_EuhNptj-qrzuZ4RifhTuF) for an understanding of that setup process. If you get stuck, just post in our #customer-support-chat Slack channel to start and if we get stuck as a team, we will see if our CE counterparts can help by asking them in #ce-chat.
```

Comment:
I might be misunderstanding something here; but, why do you recommend the k8s walkthrough video if they are setting up docker compose?  
Oh ... maybe this is just saying watch that video so you get an understanding of how to set up in k8s?  i think that's it.  :). if so just ignore this comment

Reply:
Yup! That can be loads clearer. By adding one word! Updated.